src/inputs/search/query-search/type.ts

```diff
@@ -87,9 +87,16 @@ export interface QuerySearchProps {
     keyItemSets: KeyItemSet[];
     valueHandlerMap: ValueHandlerMap;
     value: string;
-    preLimitOperators?: OperatorType[];
 }
 
 export interface QuerySearchEventArgs {
     search: [QueryItem];
 }
+
+export interface MenuFormatterArgs {
+    resp: HandlerResponse;
```

Comment:
Personally, res was more intuitive than resp!

Reply:
If it is a value that comes in as an argument of a function, I think that there is no difficulty in understanding it even if it is used as an abbreviation because the variable called `resp` is a local variable.

However, if it is the property name of the argument, it would be better to clarify what it means. What about names like `response` or `data` rather than `resp`?

---

함수의 인자로 들어오는 값이라면 resp 라는 변수가 지역변수이기 때문에 축약어로 사용해도 이해하는 데에 큰 어려움이 없다고 생각합니다. 

하지만 인자의 property 명이라면 그것이 무엇을 의미하는지 더 명확히 하는 것이 좋을 것 같습니다. resp 보다는 response 혹은 data와 같은 이름은 어떨까요?